tests/ui/conftest.py

```diff
@@ -26,12 +78,19 @@ def capabilities(capabilities):
 
 @pytest.fixture
 def fxa_account(my_base_url):
+    """
+        Account used to login to the AMO site.
+        :returns: :py:class: '~fxapom.fxapom.FxATestAccount'
+    """
     url = DEV_URL if 'dev' or 'localhost' in my_base_url else PROD_URL
     return FxATestAccount(url)
 
 
 @pytest.fixture(scope='session')
 def jwt_issuer(base_url, variables):
+    """
+    :returns: JWT Issuer from variables file or env variable named 'JWT_ISSUER'
```

Comment:
Nit: We're not generating documentation from these docstrings, so I'm not sure we need `:returns:`. Also, the first line should be a summary. In this case I think a docstring such as `"""Return the JWT issuer from variables file or 'JWT_ISSUER' environment variable."""` would be enough.